### PR TITLE
PP-11548: Remove ecs query code

### PR DESCRIPTION
**Pull request by jfharden**
With the release of https://github.com/alphagov/pay-infra/pull/4519 and https://github.com/alphagov/pay-adot/pull/9 we are now populating the default image tags in the adot collector instead of in our apps. This drastically simplifies our app metrics code. The same PR will be made for pay-java-metrics later

### CODE REVIEW

File: src/test/index.test.ts [hunk 1]
```diff
@@ -1,20 +1,11 @@
...
 
-const metricsConfig: MetricsConfigurationOptions = {
-  defaultMetricsLabels: {
-    one: 'test1',
-    two: 'test2',
-    three: 'test3',
-  },
-}
-
 jest.mock('node:http', () => ({
```
Comment (nlsteers) [inline src/test/index.test.ts:9]: this mock can be removed now that the client doesn't make any http requests

File: src/test/index.test.ts [hunk 2]
```diff
@@ -44,11 +35,10 @@ describe('/metrics endpoint', () => {
     app = express()
     metrics = require('../index')
     env.NODE_ENV = 'development'
```
Comment (nlsteers) [inline src/test/index.test.ts:37]:
```suggestion
    metrics = require('../index')
    app.use(metrics.initialise())
    env.NODE_ENV = 'development'
```

File: src/test/index.test.ts [hunk 3]
```diff
@@ -44,11 +35,10 @@ describe('/metrics endpoint', () => {
...
-    env.ECS_CONTAINER_METADATA_URI_V4 = undefined
   })
 
   it('should return metrics as plain text by default', async () => {
-    app.use(metrics.initialise(metricsConfig))
+    app.use(metrics.initialise())
```
Comment (nlsteers) [inline src/test/index.test.ts:41]:
```suggestion
```

File: src/test/index.test.ts [hunk 4]
```diff
@@ -58,7 +48,7 @@ describe('/metrics endpoint', () => {
   })
 
   it('should return metrics as json when Accept header is set', async () => {
-    app.use(metrics.initialise(metricsConfig))
+    app.use(metrics.initialise())
```
Comment (nlsteers) [inline src/test/index.test.ts:51]:
```suggestion
```

File: src/test/index.test.ts [hunk 5]
```diff
@@ -76,8 +66,8 @@ describe('/metrics endpoint', () => {
     })
   })
 
-  it('should return custom metrics with default and custom labels', async () => {
-    app.use(metrics.initialise(metricsConfig))
+  it('should return custom metrics with custom labels', async () => {
+    app.use(metrics.initialise())
```
Comment (nlsteers) [inline src/test/index.test.ts:70]:
```suggestion
```

File: src/test/index.test.ts [hunk 6]
```diff
@@ -88,15 +78,13 @@ describe('/metrics endpoint', () => {
...
-      'test_histogram_bucket{le="10",test_type="HISTOGRAM",one="test1",two="test2",three="test3"} 1'
-    )
+    expect(response.text).toContain('test_counter{test_type="COUNTER"} 1')
+    expect(response.text).toContain('test_gauge{test_type="GAUGE"} 24')
+    expect(response.text).toContain('test_histogram_bucket{le="10",test_type="HISTOGRAM"} 1')
   })
 
   it('should return histogram with custom buckets', async () => {
-    app.use(metrics.initialise(metricsConfig))
+    app.use(metrics.initialise())
```
Comment (nlsteers) [inline src/test/index.test.ts:87]:
```suggestion
```

File: src/test/index.test.ts [hunk 7]
```diff
@@ -107,19 +95,13 @@ describe('/metrics endpoint', () => {
...
-      'test_histogram_bucket{le="40",test_type="CUSTOM_BUCKETS",one="test1",two="test2",three="test3"} 1'
-    )
+    expect(response.text).toContain('test_histogram_bucket{le="20",test_type="CUSTOM_BUCKETS"} 0')
+    expect(response.text).toContain('test_histogram_bucket{le="30",test_type="CUSTOM_BUCKETS"} 1')
+    expect(response.text).toContain('test_histogram_bucket{le="40",test_type="CUSTOM_BUCKETS"} 1')
   })
 
   it('should return express http metrics', async () => {
-    app.use(metrics.initialise(metricsConfig))
+    app.use(metrics.initialise())
```
Comment (nlsteers) [inline src/test/index.test.ts:104]:
```suggestion
```

**Review by nlsteers — COMMENTED** [review]
Looks good.

Couple of suggestions to DRY up the tests now that some features are no longer required

File: src/test/index.test.ts (hunk 1 above)
Comment (jfharden) [reply src/test/index.test.ts]: 👍

**Comment by jfharden** [comment]
@nlsteers I've resolved all the suggestions 👍 Thanks for the suggestions :)

**Review by nlsteers — APPROVED** [review]
eviscerated 🔥

**Pull request merged**